### ISSUE TITLE
How can I add ```pydantic-core``` recipe

### ISSUE THREAD

**Issue by Kacper-OS** [kivy/python-for-android#2955]
I want to add ```pydantic-core``` recipe but, half of ```pydantic-core```'s code is using rust programming lanugage, and I don't know if it:  
```py
from pythonforandroid.recipe import CompiledComponentsPythonRecipe

class PydanticcoreRecipe(CompiledComponentsPythonRecipe):
    version = '2.16.1'
    url = 'https://files.pythonhosted.org/packages/2f/ae/b230a03d5500bfce92d556b993dd6911ac0463b2468d345fb1b3b05eae79/pydantic_core-{version}.tar.gz'
    depends = ['typing-extensions']

recipe = PydanticcoreRecipe()
``` 
will be enought, or do I need to add rust program to the ```pydantic-core```'s recipe folder

**Comment by baseplate-admin** [comment]
@Julian-O , i think this is a recipe request.

---

> will be enought, or do I need to add rust program to the pydantic-core's recipe folder

I dont think it's this easy to compile a rust binary.. You can try to see if CI/CD is passing, but i think it involves some direct involvement from the maintainers of `pydantic-core`

**Comment by Julian-O** [comment]
@baseplate-admin: I don't think the OP is asking us to add a recipe, but (far more excitingly) how *they* can write a recipe. Almost a support request, but for development not fixing their own bugs.

Good luck, @Kacper-OS. I have no expertise to help you with, but appreciate you putting in the effort.

**Comment by T-Dynamos** [comment]
This can be closed now.

**Comment by misl6** [comment]
Closed via #2962
